### PR TITLE
Create client-settings.md

### PR DESCRIPTION
**Pull request by TinaUll**
Hi @rgralakosi, I drafted the MQTT client settings configuration topic. Please take a look (especially the parameter descriptions; default values are missing!) and let me know if this works. Thank you and Happy Friday!

### CODE REVIEW

File: content/configuration/client-settings.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## MQTT client settings parameters
+
+The following parameters are available for configuring MQTT client settings:
+
+| Parameter     | Required | Type | Description |
+|---------------|----------|------|-------------|
+| **KeepAlivePeriod** | Optional | `string`<br>(timespan)          | The maximum time that can elapse between sending two data items. <br><br> Allowed value: >= `0:00:05` (5 seconds)<br>Default value: ``
+| **QosLevel** | Optional |  `integer`<br><br>or<br><br>`enum`              | Defines the guarantee of delivery for a specific item.<br><br> Allowed value:<br>`0` or `AtMostOnce`: There is no guarantee of item delivery <br>`1` or `AtLeastOnce`: An item is delivered at least one time<br>`2` or `ExactlyOnce`: Each item is received only one time.<br>Default value: `` 
+| **MaxInternalQueueSize**      | Optional | `integer` | Maximum number of items that can be in the adapter internal queue. <br><br>Allowed value: >= `250`<br>Default value: `` |
```
Comment (rgralakosi) [inline content/configuration/client-settings.md:57]: Zdenek changed the parameter name from **MaxInternalQueueSize** to **MaxClientQueueSize** :-)

File: content/configuration/client-settings.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## MQTT client settings example
+
+```json
+{
+    "keepAlivePeriod": "0:00:35",
+    "qosLevel": "AtLeastOnce",
+    "maxInternalQueueSize": 1000
```
Comment (rgralakosi) [inline content/configuration/client-settings.md:65]: As mentioned above, this has been changed to **maxClientQueueSize**!

File: content/configuration/client-settings.md (hunk 1 above)
Comment (rgralakosi) [inline content/configuration/client-settings.md:57]: The default value for MaxClientQueueSize is 1000.

File: content/configuration/client-settings.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Linux: `/opt/OSIsoft/Adapters/MQTT/Schemas`
+
+## MQTT client settings parameters
+
+The following parameters are available for configuring MQTT client settings:
+
+| Parameter     | Required | Type | Description |
+|---------------|----------|------|-------------|
+| **KeepAlivePeriod** | Optional | `string`<br>(timespan)          | The maximum time that can elapse between sending two data items. <br><br> Allowed value: >= `0:00:05` (5 seconds)<br>Default value: ``
+| **QosLevel** | Optional |  `integer`<br><br>or<br><br>`enum`              | Defines the guarantee of delivery for a specific item.<br><br> Allowed value:<br>`0` or `AtMostOnce`: There is no guarantee of item delivery <br>`1` or `AtLeastOnce`: An item is delivered at least one time<br>`2` or `ExactlyOnce`: Each item is received only one time.<br>Default value: ``
```
Comment (rgralakosi) [inline content/configuration/client-settings.md:56]: The default value for QosLevel is 2 ("ExactlyOnce")

File: content/configuration/client-settings.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Linux: `/opt/OSIsoft/Adapters/MQTT/Schemas`
+
+## MQTT client settings parameters
+
+The following parameters are available for configuring MQTT client settings:
+
+| Parameter     | Required | Type | Description |
+|---------------|----------|------|-------------|
+| **KeepAlivePeriod** | Optional | `string`<br>(timespan)          | The maximum time that can elapse between sending two data items. <br><br> Allowed value: >= `0:00:05` (5 seconds)<br>Default value: ``
```
Comment (rgralakosi) [inline content/configuration/client-settings.md:55]: The default value for KeepAlivePeriod is "00:00:30" (30 seconds).

**Review by rgralakosi — CHANGES_REQUESTED** [review]

**Comment by TinaUll** [comment]
Thanks, @rgralakosi. Topic is updated. Let me know if you approve now :)

**Pull request merged**